include/zephyr/drivers/i3c.h

```diff
@@ -56,19 +56,71 @@ extern "C" {
  *   - 0: No Limitation
  *   - 1: Limitation obtained via GETMXDS CCC.
  */
+
+/**
+ * @brief Bus Characteristic Register (BCR) - Max Data Speed Limitation bit.
```

Comment:
If you go for the suggestion above, you may then want to simplify as the following:

```suggestion
 * @brief Max Data Speed Limitation bit.
```

Reply:
Changed.